.golangci.yml

```diff
@@ -0,0 +1,17 @@
+issues:
+        max-same-issues: 0
+        exclude-use-default: false
+linters:
+        enable:
+        - whitespace
+        - gosec
+        - gci
+        - misspell
+        - gomnd
+        - gofmt
```

Comment:
personally, I would replace gofmt+gci+whitespace with gofumpt, which does more than all three combined too :wink: 

it will also handle long lines soon, so that would also make lll redundant.